submission_rules.adoc

```diff
@@ -21,70 +21,91 @@ Unless otherwise stated all rules apply to closed and open.
-
+The review committee is chaired by the relevant results working group chair
+unless a review chair is selected.
 
 ### Meetings
 
-The review committee makes decisions during properly scheduled meetings. The base meeting schedule is dictated by the review process. The review committee chair may schedule additional meetings during the review process in the course of any meeting or by emailing all committee members (e.g. mlperf-chairs mailing list) at least 24 hours in advance.
+The review committee makes decisions during properly scheduled meetings. The
+base meeting schedule is dictated by the review process. The review committee
+chair may schedule additional meetings during the review process in the course
+of any meeting or by emailing all committee members at least 24 hours in
+advance.
 
-The review committee has a quorum if it includes an eligible chair and at four other eligible members.
-
-Review committee meetings are typically held at a specific location determined by the review committee chair and in-person attendance is encouraged, but virtual attendance is to be enabled and allowed. 
-
-Review committee meetings are open only to the review committee and all submitters in the current submission round.
+Review committee meetings are typically held virtually.
 
+Review committee meetings are open only to the review committee.
 
 ### Agenda and decisions
 
-The review committee agenda is set by, and decisions are tracked, using issues filed against the Github submission repo. In general, issues must be filed as dictated by the review process schedule. Exceptions are discouraged but may be allowed during a meeting by a vote of the review committee. In general, issues are decided in the order filed, but the chair may choose a different order if circumstances warrant.
+The review committee agenda is set by, and decisions are tracked, using issues
+filed against the Github submission repo. In general, issues must be filed as
+dictated by the review process schedule. Exceptions are discouraged but may be
+allowed during a meeting by a vote of the review committee.
 
-The review committee should attempt to decide issues through discussion and grudging consensus whenever possible. However, if the review committee and all submitters are unable to reach a grudging consensus, the review committee will vote to decide the issue.
+The review committee should attempt to decide issues through discussion and
+grudging consensus whenever possible. However, if the review committee is unable
+to reach a grudging consensus, the submitters will vote to decide the
+issue. Non-submitters may not vote, except in the case of a tie. In the case of
+a tie, the first non-submitter from the following list will cast the deciding
+vote: the review chair, the chairs of the relevant working group shortest tenure
+first, the chairs of the relevant power working group shortest tenure first, and
+a random number generator.
```

Comment:
how about adding "the executive director and the president of the board are not allowed in the voting" just for completeness?

Reply:
Done.